simple/stats/config.md

```diff
@@ -73,12 +93,20 @@ Example:
 
 ### Input file parameters
 
+#### `importType`
+
+This importer supports importing observations, events and entities.
+The type of import can be specified with the `importType` parameter.
+Possible values: `observations` (default), `events`, `entities`.
```

Comment:
should the importType be "Entity", "Observation" and "Event"?

Reply:
Don't want to break backward compatibility. So going with what we have.

Reply:
As discussed, we should handle both and document the new/desired one.